ArduCopter/motor_test.cpp

```diff
@@ -191,7 +205,12 @@ void Copter::motor_test_stop()
     g.failsafe_throttle.load();
     g.failsafe_battery_enabled.load();
     g.failsafe_gcs.load();
+    g.fs_ekf_action.load();
 
+    if (motor_test_throttle_type == MOTOR_TEST_COMPASS_CAL) {
+        compass.per_motor_calibration_end();
```

Comment:
Nitpicking: logic-wise it would be best to do this before re-enabling the failsafes.